### PR TITLE
Clarified some language around keyspace ranges.

### PR DESCRIPTION
**Pull request by thatnerd**
* Clarifying our language around the initial range split.
* Using this commit to make sure I understand what a range
constitutes.

### CODE REVIEW

File: _includes/v19.1/misc/basic-terms.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Term | Definition
...
+**Range** | CockroachDB stores all user data (tables, indexes, etc.) and almost all system data in a giant sorted map of key-value pairs. This keyspace is divided into "ranges", contiguous chunks of the keyspace, so that every key can always be found in a single range.<br><br>From a SQL perspective, a table and its secondary indexes initially map to a single range, where each key-value pair in the range represents a single row in the table (also called the primary index because the table is sorted by the primary key) or a single row in a secondary index. As soon as that range reaches 64 MiB in size, it splits into two ranges. This process continues for the child ranges as the table and its indexes continue growing.
```
Comment (jseldess) [inline _includes/v19.1/misc/basic-terms.md]: I like that this is more explicit, but I'm a little concerned about the term `child`. There is no parent/child relationship, really, when it comes to ranges for table data. How about we just change `for the child ranges` to `for these ranges`?

**Comment by thatnerd** [comment]
This seems to have changed a lot. Going to rebase to master before anything else.

File: _includes/v19.1/misc/basic-terms.md (hunk 1 above)
Comment (thatnerd) [reply _includes/v19.1/misc/basic-terms.md]: I agree with your assessment.

**Comment by thatnerd** [comment]
Killing this PR. I'll submit changes from a branch once I have write access.

**Pull request closed without merging**